geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java

```diff
@@ -203,6 +203,29 @@ public TabularResultModel addTable(String namedSection) {
     return section;
   }
 
+  public TabularResultModel addTable(String namedSection, List<CliFunctionResult> functionResults,
```

Comment:
I don't particularly like the side effect of status setting in this method but I'm not sure what a good alternative would be. I think for now, could the method name be changed to something like `addTableSettingStatus`?